src/configure.ac

```diff
@@ -402,6 +403,11 @@ dnl AC_CHECK_SIZEOF([int *function()]
 # See http://ftp.netbsd.org/pub/NetBSD/NetBSD-release-6/src/external/bsd/pcc/dist/pcc/configure.ac for many alternatives
 # *****************************************************************
 #
+if test "x$simd" = xno ; then
+[echo "SIMD logic... DISABLED by request"]
+AC_SUBST([JOHN_SIMD_OFF],["-DJOHN_SIMD_OFF"])
```

Comment:
Actually I'd also rather have this as `JOHN_NO_SIMD` or `JOHN_DISABLE_SIMD` but I can live with this. You even accidentally used `JOHN_NO_SIMD` yourself in 69afa080 so I take it that name might be more logical for yourself too 😆

Reply:
I agree that JOHN_NO_SIMD is probably the right choice.  it is easy to fix, and now that I think we have a better working version (although not done), I am not averse to fixing the nits in it.